README.md

```diff
@@ -66,6 +66,56 @@ nexusPublishing {
 }
 ```
 
+#### Configure Signing ####
```

Comment:
Maybe add a link to https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signing_publications?

Reply:
Added manually.